src/Orleans.Runtime/Configuration/Options/SiloMessagingOptions.cs

```diff
@@ -76,5 +76,13 @@ public class SiloMessagingOptions : MessagingOptions
         /// For test only - Do not use in production
         /// </summary>
         public bool AssumeHomogenousSilosForTesting { get; set; } = false;
+
+        public static TimeSpan DEFAULT_REROUTE_QUEUED_MESSAFES_On_SILO_SHUTDOWN_TIMEOUT { get; set; } = TimeSpan.FromSeconds(10);
```

Comment:
Typo: "MESSAFES", Also the name seems too long. Suggest "DEFAULT_SHUTDOWN_REROUTE_TIMOUT" instead.